src/go/k8s/pkg/resources/certmanager/pki.go

```diff
@@ -34,6 +35,10 @@ const (
 	AdminClientCert = "admin-client"
 	// RedpandaNodeCert cert name - node certificate
 	RedpandaNodeCert = "redpanda"
+	// AdminAPIClientCert cert name - client certificate for Admin API
+	AdminAPIClientCert = "admin-api-client"
```

Comment:
nit: should we move these to the admin go file or do you think it's better here?